src/common/UserObjectType.js

```diff
@@ -0,0 +1,6 @@
+export type UserObjectType = {
+  username: ?string,
+  email: ?string,
+  id: ?string,
+  profile: ?Object,
```

Comment:
Consider adding generics to this type to give the user an option to have an extensible UserObject.